### DISCUSSION TITLE
Is it possible to run multiple commands that dont exit?

### DISCUSSION THREAD

**Discussion by drewbietron** [vercel/turborepo#2910] [Help] [answered]
I am trying to set up some dev commands that run multiple dev servers locally and I am having trouble getting them to run within a single terminal window if one of the commands doesn't exit.  

For example, I am trying to forward Stripe webhooks to local server, but that command doesn't exist.  So using it as a `dependsOn` within a dev command won't run the subsequent commands, it'll just hang on the first command since it never exits.  
```bash
stripe listen --forward-to localhost:54321/functions/v1/stripe-webhooks
```

I've been getting around this by just opening different windows and running the commands that don't exit, or putting the non-existing command last in the chain of commands.  It would be really slick if with one command I can could spin up the various services, have them listen on their ports or whatever it is they do if they dont exit, and still have the command finish every other command.

Any help would be appreciated!

**Comment by mehulkar** [comment, marked as answer]
It's not possible right now to _depend on_ a task that doesn't exit. We introduced [a config to make this harder to do](https://turbo.build/blog/turbo-1-7-0#schedule-your-long-running-tasks-with-confidence) in v1.7. 

You can run non-exiting tasks with the `--parallel` flag, but if you do, they will not be aware of each other, and will not wait for one to be available before the other one tries to run.
